localScrapper.js

```diff
@@ -8,9 +8,14 @@ const isVideoExtension = extension => [
 function getFileInfos(fileName, dirPath, stat) {
   const lastIndexOfSlash = dirPath.lastIndexOf('/')
   const dirName = lastIndexOfSlash !== -1 ? dirPath.substr(lastIndexOfSlash + 1) : '.'
-  const fileNamePatterns = fileName.match(/^(.+) - ([A-Z0-9 ]+) - (.+)\.(.{2,4})$/) || []
-  const [, group, type, songName, extension] = fileNamePatterns
+  const fileNamePatterns = fileName.match(/^(.+) - ([A-Z0-9 ]+) - (.+) ?(\(.+\))?\.(.{2,4})$/) || []
```

Comment:
Then again, after consulting with the boss, there is only one language associated with a file, and it will only be 3 letters, so we could add that to the regexp.